src/platform/mock-local-home-platform.ts

```diff
@@ -66,9 +68,12 @@ export class MockLocalHomePlatform {
   public async triggerIdentify(discoveryBuffer: Buffer): Promise<void> {
     console.log('received discovery payload:', discoveryBuffer);
 
-    if (this.app == null) {
+    if (this.app === undefined) {
+      throw new Error('Cannot trigger IdentifyRequest: App was undefined');
+    }
+    if (!this.isHomeAppReady()) {
```

Comment:
Are you going to write tests to check these errors are thrown correctly?

Reply:
Yeah, I plan to add tests soon ( Re: #31), unless you think I should just go ahead and do it here.

Reply:
Generally you add tests at the same time as behavior to ensure your behavior works as expected.

Reply:
Will add

Reply:
Done